scipy/spatial/distance.py

```diff
@@ -236,6 +238,13 @@ def directed_hausdorff(u, v, seed=0):
     result = _hausdorff.directed_hausdorff(u, v, seed)
     return result
 
+
+def _validate_aitchison(w, u_v):
```

Comment:
Parameters of ``aitchison`` are ``u`` and ``v``, so these parameter names are confusing (especially since ``w`` is a parameter to ``pdist``/``cdist``). How about ``u_or_v, uv_name``?